DragAndDrop/sample/src/main/java/com/microsoft/device/dualscreen/draganddrop/ui/theme/Shape.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.draganddrop.ui.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
+    small = RoundedCornerShape(4.dp),
+    medium = RoundedCornerShape(4.dp),
+    large = RoundedCornerShape(0.dp)
```

Comment:
can update large shape here for use in multiple places!
```suggestion
    large = RoundedCornerShape(20f)
```